post.py

```diff
@@ -9,51 +9,30 @@
 
 ###--------------------store function--------------------------###
 
-def getForces(source_path):
+def getForces(source_path, force_keys=["fpx", "fpy", "fpz", "fvx", "fvy", "fvz"]):
```

Comment:
Das macht überhaupt keinen Sinn, da die Labels immer so bleiben. Die werden ja von openFOAM in die output Datei als header geschrieben.